src/provenanceDb.ts

```diff
@@ -70,6 +75,18 @@ export async function getWorkflowStatuses(debugConfig: DbosDebugConfig, method?:
   } finally {
     client.release();
   }
+
+  async function getPoolClient(): Promise<PoolClient> {
+    try {
+      return await getPool(debugConfig).connect();
+    } catch (e) {
+      if (isPgError(e) && e.code === '3D000') {
+        throw new Error(`Provenance database does not exist. This app was not registered with --enable-timetravel.`);
```

Comment:
```suggestion
        throw new Error(`Provenance database does not exist. This app was not deployed with --enable-timetravel.`);
```
We're going to eliminate the register command in our next release. Everything is in the `deploy` command now.